spec/features/user_visits_index_page_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'user visits index page' do
+
+  let!(:gong_cha) { Shop.create(name: "Gong Cha", address: "40-44 Harrison Ave", city: "Boston", state: "MA", zip: "02111") }
+  let!(:onezo) { Shop.create(name: "OneZo", address: "83 Harrison Ave", city: "Boston", state: "MA", zip: "02111") }
+
+  scenario 'user sees list of bubble tea shops' do
+    visit "/"
+
+    expect(page).to have_content "Gong Cha"
+    expect(page).to have_content "OneZo"
+  end
+
+  scenario 'user clicks on a shop' do
+    visit "/"
+    click_link "Gong Cha"
+
+    expect(page).to have_content gong_cha.name
+    expect(page).to have_content gong_cha.address
+    expect(page).to have_content gong_cha.city
+    expect(page).to have_content gong_cha.state
+    expect(page).to have_content gong_cha.zip
+  end
```

Comment:
this last scenario is technically a show page spec....which you will be deleting

but you can still have test check that line 17 works, since that will still be in Rails/erb